src/main/java/com/unosquare/admin_core/back_end/dto/ClientDTO.java

```diff
@@ -6,6 +6,5 @@
 public class ClientDTO {
 
     private int clientId;
-
     private String clientName;
-}
+}
```

Comment:
Try to always add a newline to the end of each file. A lot of projects will have some sort of warning / error with Checkstyle for this.
https://stackoverflow.com/questions/4700312/empty-new-line-at-the-end-of-the-java-source-files